test_aodndata/moorings/test_mooringsProductsHandler.py

```diff
@@ -5,27 +5,34 @@
 
 from aodncore.pipeline import (PipelineFile, PipelineFileCollection,
                                PipelineFilePublishType)
+from aodncore.pipeline.exceptions import InvalidFileContentError
 from aodncore.pipeline.storage import get_storage_broker
 from aodncore.testlib import HandlerTestCase, make_test_file
 
 from aodndata.moorings.products_handler import MooringsProductsHandler, MooringsProductClassifier, get_product_type
 
 TEST_ROOT = os.path.dirname(__file__)
 GOOD_MANIFEST = os.path.join(TEST_ROOT, 'test_product.json_manifest')
+AGGREGATED_ONLY_MANIFEST = os.path.join(TEST_ROOT, 'test_product_aggregated.json_manifest')
+BAD_VAR_MANIFEST = os.path.join(TEST_ROOT, 'test_product_bad_var.json_manifest')
 PRODUCT_FILE = os.path.join(
```

Comment:
wouldn't file name parameters be best to be arguments in a test function!?

Reply:
Ahh nevermind - you are loading cases here - better than put a little comment above.